src/features/device-logs/lib/backends/loki.ts

```diff
@@ -198,6 +198,12 @@ export class LokiBackend implements DeviceLogsBackend {
 			gzip: LOKI_HISTORY_GZIP,
 		});
 
+		if (statusCode !== 200) {
+			throw new Error(
```

Comment:
@Page- will this unhandled error show up on sentry (e.g. generic type Error rather than a properly HttpError?)

Reply:
Yes, every path that makes it here has a generic error handler with reporting to sentry, this is mostly just changing the error from
```
     TypeError: Cannot read properties of undefined (reading 'result')
      at LokiBackend.history (file:///usr/src/app/src/features/device-logs/lib/backends/loki.ts:129:28)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async Context.<anonymous> (file:///usr/src/app/test/13_loki-backend.ts:102:29)
```
to the actual error returned from loki